templates/data-sources/metal_connection.md.tmpl

```diff
@@ -20,7 +20,7 @@ Use this data source to retrieve a [connection resource](https://metal.equinix.c
 
 The following arguments are supported:
 
-* `connection_id` - (Required) ID of the connection resource.
+{{ .SchemaMarkdown | trimspace }}
 
 ## Attributes Reference
```

Comment:
This `## Attributes Reference` can also be removed; it's part of the `.SchemaMarkdown` output